src/widgets_som_parser.js

```diff
@@ -83,16 +84,19 @@ class WidgetsSOMParser {
   }
 
   parseEmail(data) {
+    console.log("HOWDY1****");
```

Comment:
don't forget to take these "Howdy" statements out at some point - also not sure what happens in IE.

Reply:
I just did...Thanks for pointing it out..